packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -89,18 +93,16 @@ function App() {
           setTelemetryProperties(properties);
 
           try {
-            const deviceId = await getUID();
-
-            await fetch('https://analytics.strapi.io/track', {
+            fetch('https://analytics.strapi.io/api/v2/track', {
```

Comment:
```suggestion
            await fetch('https://analytics.strapi.io/api/v2/track', {
```

Otherwise the rejected promise can not be fetched, which leads to an error will crash the whole UI.

Reply:
Funnily enough, this is not my code and it has been working for all the time now 😅

Reply:
No it didn't :) I recently fixed all of these instances ...

Reply:
But I merged master into this branch before pushing. I should have had all these changes if you fixed them already 🤔

Reply:
You should. Did you merge `master` or `main`? The change was there, so maybe it got lost in a merge conflict? Some of the other changes (the ones to `useTracking()` are there).

Reply:
Main*. My bad. I've fixed all merge conflicts, had only one with await in the `useTracking()`. Didn't have any others